metrics/default_metrics/tarantool/info.lua

```diff
@@ -0,0 +1,31 @@
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_info_metrics()
```

Comment:
Maybe update_info_metrics instead of get?

There is confusion with current naming as it seems the function returns memory_metrics. But it just updates / creates some gauges.